CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # avro-schema-registry
 
+## v0.11.0
+- Change the default fingerprint version to '2'.
```

Comment:
Should we tell users to explicitly set `FINGERPRINT_VERSION` to `1` before upgrading if they haven't already migrated to fingerprint version 2?

Reply:
Yes, I can add that. I'll also include a note in the README.